src/main/java/org/javamodularity/moduleplugin/tasks/StartScriptsMutator.java

```diff
@@ -108,6 +109,20 @@ private void updateRelativePath(FileCopyDetails fileCopyDetails) {
         fileCopyDetails.setRelativePath(updatedRelativePath);
     }
 
+    private void updateFinalRunCmnd(CreateStartScripts startScriptsTask) {
+        System.out.println("Ok, replacing here....");
```

Comment:
Do you really need this line or it is just a debug leftover?

Reply:
Sigh... Fixed 👍